public/pages/ReviewAndCreate/containers/ReviewAndCreate.tsx

```diff
@@ -72,11 +80,70 @@ export function ReviewAndCreate(props: ReviewAndCreateProps) {
   const dispatch = useDispatch();
   useHideSideNavBar(true, false);
 
+  const [validDetectorSettings, setValidDetectorSettings] = useState(false);
+  const [validModelConfigurations, setValidModelConfigurations] =
+    useState(false);
+  const [validationError, setValidationError] = useState(false);
+  const [settingsResponse, setDetectorMessageResponse] =
+    useState<validationSettingResponse>({} as validationSettingResponse);
+  const [featureResponse, setFeatureResponse] =
+    useState<validationModelResponse>({} as validationModelResponse);
+  const [isCreatingDetector, setIsCreatingDetector] = useState(false);
+  const isLoading = useSelector((state: AppState) => state.ad.requesting);
+
   // Jump to top of page on first load
   useEffect(() => {
     scroll(0, 0);
   }, []);
 
+  useEffect(() => {
+    dispatch(validateDetector(formikToDetector(props.values)))
+      .then((resp: any) => {
+        if (!Object.keys(resp.response).length) {
+          setValidDetectorSettings(true);
+          setValidModelConfigurations(true);
+        } else {
+          if (resp.response.hasOwnProperty('detector')) {
+            const issueType = Object.keys(resp.response.detector)[0];
```

Comment:
can this be undefined?

Reply:
also, what if there's multiple issues? Seems this would only store one issue

Reply:
Based on backend code there is no possible case where response doesn't have property `'detector'` if it fails previous check of no keys in response. This however will change to `'detector'` or `'model'` for non-blocker release. I can add another check to see if undefined just for safety.

Currently there is only one issue found per validation response unless categorized as a sub_issue of a larger issueType. Validation checks are done sequentially so exact root cause of issue can be found. In current design user will have to fix validation issue one at a time since one issue can be root cause for multiple issues if we do all checks at once.

Reply:
I see. What about cases where there is 2 blocking issues that don't have dependencies on each other fail? For example, invalid detector name and invalid index? Understood this is a backend limitation, just wondering.

I'm assuming with non-blocker, there will be scenarios where multiple failures may happen, can revisit frontend handling when that is done. I'm ok with this for now.

Reply:
There is currently no logic to show 2 blocking issues when they don't have a specific dependency on each other. Initial design review led to conclusion that making it sequential the same way we do with create/update is cleaner as those codes can now be nicely coupled. I can revisit this however and try to see if changes could be made for all validate/create/update in terms of detecting two block issues that don't have dependency for each other. And yes will definitely revisit in terms of non-blocker implementation.

Reply:
Cool, no concerns for now.